docs/tutorials/change-logo.md

```diff
@@ -50,7 +49,7 @@ Finally we will upload the favicon to your site repository.
 1. In a browser go to your site repository.
 1. In the list of files, navigate to the `assets/img/favicons` folder.
 1. Click `Add a file` and then `Upload files`.
-1. From the unzipped folder of images, drag in both of the files named `favicon`, or click to browse for them.
+1. From the unzipped folder of images, drag in all of the images. Or click to browse for them and select all the images from that unzipped folder.
```

Comment:
Just to double check is it just the .png files from the unzipped folder i.e. not the favicon.ico and site.webmanifest?

Reply:
@LucyGwilliamAdmin Good point - to avoid ambiguity I just changed it to "all of the files". They're not all technically required but it doesn't hurt to add them.